catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -297,10 +308,24 @@ public CatalogFrameworkImpl(BundleContext context, CatalogProvider catalogProvid
             QueryResponsePostProcessor queryResponsePostProcessor, ExecutorService pool,
             SourcePoller poller, ResourceCache resourceCache,
             DownloadsStatusEventPublisher eventPublisher, ReliableResourceDownloadManager rrdm) {
-        this(Collections.singletonList(catalogProvider), context, preIngest, postIngest, preQuery,
-                postQuery, preResource, postResource, connectedSources, federatedSources,
-                resourceReaders, queryStrategy, queryResponsePostProcessor, pool, poller,
-                resourceCache, eventPublisher, rrdm);
+        this(Collections.singletonList(catalogProvider),
```

Comment:
Agree. This constructor has been deprecated and might be able to be removed in the refactoring ticked. The change to this constructor is only from the formatter. See the CatalogFrameworkImpl(FrameworkProperties) for the currently supported constructor.